docs/setup.md

```diff
@@ -0,0 +1,119 @@
+
+### Install MySQL Server and MySQL Workbench
+Rasa.NET uses MySQL to store game data. Download and install MySQL Community Server following these steps:
+
+- Download the [MySQL Installer for Windows](https://dev.mysql.com/downloads/windows/installer/8.0.html)
+- Run the installer after it's downloaded
+- The default options are fine, but you can customize the install and choose only MySQL Sever and MySQL Workbench. 
+- For help, follow [the documentation](https://dev.mysql.com/doc/refman/5.7/en/mysql-installer-setup.html)
+- Set the `root` user password to something unique when prompted after installation
+
+### Install Git
+Git is a version control system and will allow you to download the code. Alternatively, if you're not planning on developing for Rasa.NET you can download a .zip of the code directly from GitHub.
+
+- Download [Git](https://git-scm.com/downloads)
+- Run the installer. If you're not sure of what options to select, choose the defaults
+
+## Download and install the game
+You'll need the game client. Below is the demo version which was made freely available. 
+
+- Refer to the forums to [download and install the game client](https://infiniterasa.org/viewtopic.php?f=15&t=8).
+- Create a shortcut to the game client on the desktop. You'll need to find the executable in the path where you installed the game
+  - Right-click on the `.exe` and choose `Send to > Desktop (create shortcut)`
+  - Go to the desktop and right-click on the newly created shortcut and choose `Properties`
+  - Select the `Shortcut` tab
+  - Edit the `Target` box to append `/NoPatch /AuthServer=localhost:2106` to the end of the file path. 
+
+## Patch the game using the custom patcher
+The game client needs to be patched to a specific version that is compatible with Rasa.NET. 
+
+- [Download and run the custom patcher](https://infiniterasa.org/viewtopic.php?f=15&t=8)
+
+## Download or clone the code
+Download Rasa.NET from GitHub or use Git to clone the project (recommended). 
+
+- Launch a command prompt or Git bash terminal
+- Change directories to where you want the source to download to, i.e. `cd C:\Projects`
+- Run the command `git clone https://github.com/InfiniteRasa/Rasa.NET.git`
+
+### Setup the database
+Before you can build and run the code, there are some configuration steps for the database. 
+
+- Launch MySQL Workbench and select your `Local instance MySQL80` under `MySQL Connections` that was created during install
+- Enter your root password that you created to connect to it
+- In the Navigator panel on the left, right-click in the blank space and choose `Create Schema`
+  - In the new tab, name the first schema `rasaauth` and click `Apply`
+  - Repeat this two more times for `rasachar` and `rasaworld`
+- Double-click `rasaauth` in the `SCHEMAS` list which makes it active
+- Go to `File > Open SQL Script`
+- Select `Rasa.NET\database\full\auth_database.sql` file in the code directory that you clone in the above steps
```

Comment:
If we merge it to master, they won't find the sql files, as they are on another branch yet.

Reply:
True, thanks for bringing this up. I didn't find a good commit to pull over the correct files. 

Is it adequate enough for me to pull the creation scripts into master instead?